website/docs/cdktf/index.html.md

```diff
@@ -30,9 +29,13 @@ You can use every Terraform provider and module available on the [Terraform Regi
 
 ## When to use CDK for Terraform
 
-CDK for Terraform should be used when users have a strong preference for a procedural language, or when the ability to create abstractions using Constructs is needed to help manage complexity. Users who are already familiar with HCL and do not need these capabilities should continue using HCL Terraform. This choice can be made team-by-team and project-by-project, because CDK for Terraform [interoperates with existing Terraform providers and modules](./concepts/interoperability-workflows.html)
+CDKTF offers many benefits, but it is not the right choice for every project. You should consider using CDKTF when:
 
-**TODO:** Insert content about when folks should use this instead of HCL... link to HCL interoperability page.
+-  You have a strong preference or need to use a procedural language to define infrastructure.
+- You need to create abstractions to help manage complexity. TO DO - example here
+- You are comfortable being an early adopter and do not require commercial support.
+
+You can make this choice or each team and project because CDK for Terraform [interoperates with existing Terraform providers and modules](./concepts/interoperability-workflows.html).
 
 ## Project Maturity
```

Comment:
Is this section supposed to have the copy from [the readme](https://github.com/hashicorp/terraform-cdk#project-maturity-and-production-readiness)?

Reply:
Um yes - I think it's appropriate to say here that this under active development and so I put this here. We do something similar in the plugin provider framework docs: https://www.terraform.io/docs/plugin/framework/index.html. If you disagree with having this here, let me know! Perhaps @schersh can also weigh in on this!

Reply:
@laurapacilio I think it would be best if the #Product Maturity section in docs was the same as the content we have in the README (https://github.com/hashicorp/terraform-cdk#project-maturity-and-production-readiness) the README copy has already been vetted internally, and we should be consistent with the wording in both places. Does that sound okay to you?

Reply:
My only other question is this section, is if it would be worthwhile linking to our forum & issues since we're asking for feedback here? 

Something like: 
You can drop notes in our [discussion forum](https://discuss.hashicorp.com/c/terraform-core/cdk-for-terraform/47), and report any issues you encounter in our [issue tracker](https://github.com/hashicorp/terraform-cdk/issues/new/choose).

Reply:
oh wow I didn't realize the README was official copy. I'll change it to be exactly that. And I can add a callout to the forum and issue tracker - great idea!

Reply:
@schersh FWIW I added some very similar language to the community doc https://github.com/hashicorp/terraform-cdk/pull/1066

Reply:
do you think it's overkill to have it in both places? I think it's nice to add the destination wherever we say we want feedback

Reply:
@schersh In a recent commit, I added a link to the community page Daniel made! I agree that we should provide some type of link to guide folks - and I was thinking this could be a good compromise between not having anything and repeating content that's on the community page. Check it out in the most recent draft and let me know your thoughts on this approach!

Reply:
Perfect, thanks @laurapacilio  - that sounds good to me!